apps/aehttp/src/sc_ws_handler.erl

```diff
@@ -86,7 +92,8 @@ set_channel_id_(#{channel_id := A}, #handler{channel_id = B})
 set_channel_id_(_Msg, H) ->
     H.
 
-terminate(_Reason, _PartialReq, #{} = _H) ->
+terminate(Reason, _PartialReq, #{} = _H) ->
+    lager:debug("WebSocket dying because of ~p", [Reason]),
```

Comment:
I see that terminate [does not require](https://ninenines.eu/docs/en/cowboy/2.2/guide/handlers/#_cleaning_up) trapping exit to be called. Ok.